src/EnergyPlus/WindowManager.cc

```diff
@@ -6942,8 +7169,29 @@ namespace WindowManager {
             }
         }
 
-        NominalConductance = 1.0 / (rOut + Rbare + rIn); // Autodesk:Uninit Rbare was uninitialized if ngllayer > 4
+        NominalConductance = 1.0 / (rOut + Rbare + rIn);
+        Real64 inputU = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).SimpleWindowUfactor;
+        // only compute adjustment ratio when there is valid user input U
+        if ((WinterSummerFlag == 1)) {
+            if (inputU > 0) {
+                state.dataHeatBal->CoeffAdjRatio(ConstrNum) = inputU / NominalConductance;
+                NominalConductance = NominalConductance * state.dataHeatBal->CoeffAdjRatio(ConstrNum);
+            }
+        }
         // EPTeam - again -- believe that is enforced in input //Autodesk But this routine is not self-protecting: Add as an assert
+
+        // init the surface convective and radiative adjustment ratio
+        for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
+            int const firstSurfWin = state.dataHeatBal->Zone(zoneNum).WindowSurfaceFirst;
+            int const lastSurfWin = state.dataHeatBal->Zone(zoneNum).WindowSurfaceLast;
+            for (int SurfNum = firstSurfWin; SurfNum <= lastSurfWin; ++SurfNum) {
+                if (state.dataSurface->Surface(SurfNum).ExtBoundCond == ExternalEnvironment) {
+                    int ConstrNum = state.dataSurface->Surface(SurfNum).Construction;
+                    state.dataHeatBal->SurfWinCoeffAdjRatioIn(SurfNum) = state.dataHeatBal->CoeffAdjRatio(ConstrNum);
```

Comment:
`SurfWinCoeffAdjRatioIn` and `SurfWinCoeffAdjRatioIn` can both be `dataHeatBal` variables to be consistent?

Reply:
Or should I make them both be dataHeatBalSurf variables? as SurfHConvInt and SurfHcExt are both members of dataHeatBalSurf?